src/layouts/Sidebar.tsx

```diff
@@ -92,30 +93,71 @@ const MenuItem = memo(
     onClick,
     visible = true,
   }: MenuItemProps) => {
+    const { asPath } = useRouter();
+
+    const baseUrl = publicRuntimeConfig.baseUrl;
+    const currentPath = new URL(asPath, baseUrl).pathname;
```

Comment:
Doesn't `useRouter` also expose a normal path?

Reply:
I don't remember exactly why but I wasn't able to do what I wanted with the variables exposed by useRouter.